index.rst

```diff
@@ -629,7 +629,67 @@ When you connect to the computer running a CSC and edit the configuration direct
   version: heads/tickets/DM-12345-0-g79e2257-dirty
 
 When this happens, it prevents us from precisely identifying what configuration was used.
-In this case, the preferred solution is to use :ref:`section-on-the-fly-config` to ensure traceability is not lost, at the expense of a couple extra minutes.
+In this case, the preferred solution is to follow the `On-the-Fly Configuration Changes`_ process to ensure traceability is not lost, at the expense of a couple extra minutes.
```

Comment:
"of a couple extra minutes": I suggest "of a few extra minutes".

Can we suggest or insist that the changes be committed and pushed? That would improve traceability.